docs/android/get-started/java-developers.md

```diff
@@ -814,4 +811,4 @@ study.
 - [Java Integration Overview](~/android/platform/java-integration/index.md)
 - [C# Programming Guide](/dotnet/csharp/programming-guide/)
 - [C# Reference](/dotnet/csharp/language-reference/index)
-- [Moving to C# and the .NET Framework, for Java Developers](https://www.microsoft.com/download/details.aspx?id=6073)
\ No newline at end of file
+- [Moving to C# and the .NET Framework, for Java Developers](https://www.microsoft.com/download/details.aspx?id=6073)
```

Comment:
I didn't make this change explicitly. I used the GitHub editor, so maybe it made this change w.r.t. an .editorconfig?
I can remove it if it is undesirable.